testing/api_handler.go

```diff
@@ -80,3 +101,12 @@ func cleanProducts(products []model.Product) []model.Product {
 	}
 	return products
 }
+func cleanOrders(orders []model.Order) []model.Order {
+	for i := range orders {
+		orders[i].ID = 0
```

Comment:
`orders[i] = model.Order{}` wouldn't work, but `orders[i] = model.Order{Subtotal: orders[i].Subtotal}` would. That said, doing so obscures the fact that this function is clearing nondeterministic fields. It also would introduce an area for subtle bugs in the future, because, new fields in `model.Order` should have to opt-in to being stripped, not opt-out.